.github/workflows/yarn.yml

```diff
@@ -25,30 +24,32 @@ jobs:
       sha_short: ${{ steps.vars.outputs.sha_short }}
 
     steps:
-    - uses: actions/checkout@v3
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v3
-      with:
-        node-version: ${{ matrix.node-version }}
-        cache: 'npm'
-    - name: Install production-level yarn dependencies
-      uses: Borales/actions-yarn@v3.0.0
-      with:
-          cmd: install --frozen-lockfile --production
-    - name: Build output for prod
-      uses: Borales/actions-yarn@v3.0.0
-      with:
+      - uses: actions/checkout@v3
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: "npm"
+      - name: Install production-level yarn dependencies
+        uses: Borales/actions-yarn@v3.0.0
+        with:
+          cmd: install --frozen-lockfile
+      - name: Prettier check
+        run: yarn "prettier:check"
```

Comment:
If you hide the whitespaces change (`&w=1`), this the only real change.
I removed the `--production` flag to have prettier installed, since it's a dev dependency. I'm pretty sure it doesn't have any impact on the final build, it might just make the install very slightly slower.